### PR TITLE
ci(gha): ignore duplicate commits with label presence

### PR DESCRIPTION
**Pull request by tiagolobocastro**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.

### CODE REVIEW

**Review by chriswldenyer — APPROVED** [review]
Looks reasonable

File: .github/workflows/pr-commitlint.yml [hunk 1]
```diff
@@ -36,11 +38,13 @@ jobs:
             # Ensure code-review commits don't get merged
             sed "s/code-review-rule': \[0/code-review-rule': [2/g" -i commitlint.config.js
             npx commitlint --from $first_commit --to $last_commit -V
-
             git log --pretty=format:%s $first_commit..$last_commit > ./subjects
             duplicates="$(cat ./subjects | sort | uniq -D)"
-            if [ "$duplicates" != "" ]; then
-              echo -e "Duplicate commits found:\n$duplicates" >&2
-              exit 1
+            ignore_dups="$(gh pr view ${{ github.event.pull_request.number }} --json labels -q .labels.[].name | grep -w 'ignore duplicate commits' || true)"
```
Comment (dsharma-dc) [inline .github/workflows/pr-commitlint.yml:43]: I am unaware of the conventions here, but I assume the label name here is `ignore duplicate commits`. Would it be prudent to keep label as single word without spaces, like in general labels, `ignore_duplicate_commits`

**Review by dsharma-dc — APPROVED** [review]

File: .github/workflows/pr-commitlint.yml (hunk 1 above)
Comment (tiagolobocastro) [reply .github/workflows/pr-commitlint.yml:43]: from most github labels seems convention is to use spaces

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**